ci/check-runtime-benchmarks.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eE -x;
+
+bash -c "while true; do sleep 30; echo \$(date) - running ...; done" &
+PING_LOOP_PID=$!
+trap 'kill $PING_LOOP_PID' ERR 1 2 3 6
+
+# Check if the runtime benchmarks can be compiled
```

Comment:
```suggestion
# Check if the runtime benchmarks can be compiled.
```

Reply:
Is there a plan to do more here eventually? Is it worth mentioning in a comment?

Reply:
I added a comment for that.